xml/art-obs-beginners-guide.xml

```diff
@@ -156,14 +147,15 @@ https://www.suse.com/communities/blog/suse-studio-integration/
   </para>
   <para>
    Other services around &obsa; (like the notification server, mirror
-   interface etc.) are relatively specific in purpose and are of lesser
+   interface etc.) are relatively specific in purpose and therefore of lesser
    importance for this guide.
-   <remark>toms 2017-08-16: Hmn, I think the picture does not reveal
-   "more information". </remark>
-   For more information on them, see <xref linkend="fig.obsbg.concept"/>.
+  </para>
+  <para>
+   The schematic in <xref linkend="fig.obsbg.concept"/> shows the components
+   in context.
```

Comment:
If you have a more up-to-date image, please send it to me (or replace it yourself, it's an SVG in `images/src/svg/obs-concept.svg`).